openapi.yaml

```diff
@@ -181,7 +183,45 @@ paths:
 definitions:
   ProposedEntry:
     type: object
-    $ref: "pkg/types/schemas/base_schema/v0.0.1/base_schema.json"
+    discriminator: kind
```

Comment:
this means it will use the value of `kind` to match to a different definition name in this schema to continue evaluation.